src/LaravelFacebookSdk/LaravelFacebookSdk.php

```diff
@@ -24,12 +24,23 @@ class LaravelFacebookSdk extends Facebook
     public function __construct(Config $config_handler, Url $url, array $config)
     {
         $this->config_handler = $config_handler;
-        $this->url = $url;
+        $this->url            = $url;
```

Comment:
No extra spaces between the variable and the `=` here to align it. It makes diffs harder to read.